src/Hangfire.Core/Dashboard/Pages/ProcessingJobsPage1.generated.cs

```diff
@@ -317,23 +402,23 @@ Delete selected
 
 
             
-            #line 99 "..\..\Dashboard\Pages\ProcessingJobsPage.cshtml"
+            #line 139 "..\..\Dashboard\Pages\ProcessingJobsPage.cshtml"
                                  if (job.Value.StartedAt.HasValue)
                                 {
                                     
             
             #line default
             #line hidden
             
-            #line 101 "..\..\Dashboard\Pages\ProcessingJobsPage.cshtml"
-                               Write(Html.RelativeTime(job.Value.StartedAt.Value));
+            #line 141 "..\..\Dashboard\Pages\ProcessingJobsPage.cshtml"
+                               Write(Html.Raw(job.Value.StartedAt.Value.ToString("dd/MM/yyyy HH:mm")));
 
             
             #line default
             #line hidden
             
-            #line 101 "..\..\Dashboard\Pages\ProcessingJobsPage.cshtml"
-                                                                                 
+            #line 141 "..\..\Dashboard\Pages\ProcessingJobsPage.cshtml"
+                                                                                                     ;
```

Comment:
this file is generated from the .cshtml file through the razor tool. Everything looks a bit messed up in those files :x